backend/tests/proposal/test_api.py

```diff
@@ -108,3 +108,17 @@ def test_invalid_status_proposal_publish_proposal(self):
         self.proposal.status = PENDING  # should be APPROVED
         resp = self.app.put("/api/v1/proposals/{}/publish".format(self.proposal.id))
         self.assert400(resp)
+
+    # /
+    def test_get_proposals(self):
+        self.test_publish_proposal_approved()
+        resp = self.app.get("/api/v1/proposals/")
+        self.assert200(resp)
+
+    def test_get_proposals_does_not_include_team_member_email_addresses(self):
+        self.test_publish_proposal_approved()
+        resp = self.app.get("/api/v1/proposals/")
+        self.assert200(resp)
+        for each_proposal in resp.json:
+            for team_member in each_proposal["team"]:
+                self.assertIsNone(team_member.get('email_address'))
```

Comment:
Great test 👍 We should also have one to make sure that some endpoints _do_ include emails, so that doesn't regress either.

Reply:
right on, we had the same thinking